python/paddle/fluid/tests/unittests/test_py_reader_combination.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle.fluid as fluid
+import unittest
+import numpy as np
+
+
+class TestPyReaderCombination(unittest.TestCase):
+    def setUp(self):
+        self.n1 = 10
+        self.n2 = 20
+        self.batch_size = 2
+
+    def create_reader(self, batch_num):
+        def __impl__():
+            for _ in range(batch_num):
+                image = np.random.uniform(
```

Comment:
Set fixed np.random.seed at the head of the file. Because it helps SWEs to debug when unit test error is reproducible.

Reply:
It is not necessary to fix random seed of numpy. Because we call `fluid.io.cache` to cache the reader data for the first time.

Reply:
Discussed offline, the io.cache provides same function as fixing random numbers. So this PR LGTM.